File: packages/pi-structured-return/benchmarks/test-runners/phpunit/MathTest.php
<?php

declare(strict_types=1);

use PHPUnit\Framework\TestCase;

final class MathTest extends TestCase
{
    public function testAddsTwoNumbersCorrectly(): void
    {
        $this->assertSame(2, 1 + 1);
    }

    public function testMultipliesTwoNumbersCorrectly(): void
    {
        $this->assertSame(99, 3 * 4);
    }

    public function testDoesNotDivideByZero(): void
    {
        $result = 1 / 0;
        $this->assertSame(5, $result);
    }
}

File: packages/pi-structured-return/benchmarks/test-runners/minitest/math_test.rb
require "minitest/autorun"

class MathTest < Minitest::Test
  def test_adds_two_numbers_correctly
    assert_equal 2, 1 + 1
  end

  def test_multiplies_two_numbers_correctly
    assert_equal 99, 3 * 4
  end

  def test_does_not_divide_by_zero
    result = 10 / 0
    assert_equal 5, result
  end
end
